### PR TITLE
[varLib.avar] New module to compile avar from .designspace file

### PR DESCRIPTION
**Pull request by behdad**
Fixes https://github.com/fonttools/fonttools/issues/3275

### CODE REVIEW

File: Lib/fontTools/varLib/avar.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        "designspace", metavar="family.designspace", help="Designspace file."
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file",
+        type=str,
+        help="Output font file name.",
+    )
+
+    logging_group = parser.add_mutually_exclusive_group(required=False)
```
Comment (anthrotype) [inline Lib/fontTools/varLib/avar.py:36]: mutually exclusive group but contains only one option as far as I can see..

**Review by anthrotype — APPROVED** [review]

File: Lib/fontTools/varLib/avar.py (hunk 1 above)
Comment (behdad) [reply Lib/fontTools/varLib/avar.py:36]: Humm, yeah. There wasn't a point to add `--quiet`. So I just remove the `logging_group` completely?

Comment (anthrotype) [reply Lib/fontTools/varLib/avar.py:36]: ok

Comment (behdad) [reply Lib/fontTools/varLib/avar.py:36]: Fixed.

**Pull request merged**